extendr-api/build.rs

```diff
@@ -26,4 +26,8 @@ fn main() {
     if &*major >= "4" && &*minor >= "2" {
         println!("cargo:rustc-cfg=use_r_ge_version_15");
     }
+
+    if &*major >= "4" && &*minor >= "3" {
+        println!("cargo:rustc-cfg=use_r_version_16");
```

Comment:
```suggestion
        println!("cargo:rustc-cfg=use_r_altlist");
```

Ah, sorry that I wasn't very clear. I meant you should use some meaningful name for the feature.

Reply:
I can rename it, but this isn't a feature where you *enable* `altlist`. It is more like a gate.
A `use_r_altlist` would enable it or you'd expect it to disable it.

What do you think? Let me know, and I'll change it.

Reply:
I've only written 1 feature flag before so I'm not an expert. Is there any way to enable it such that it's on by default for R >= 4.3 but disabled otherwise?

Reply:
That's what it is right now.
But the name I'd say is confusing from a "logic"/words point of view. If @yutannihilation disagrees,
then I'll rename it right away.. I initially thought it should be `use_altlist` too..

Reply:
If its okay, I'd like to have `use_r_altlist` setting it to false is clear what the decision is.

Reply:
Done